#17 Date display formatting

kopijunkie opened the pull request:
Created a dateFormatter `util` to format the API's returned datetime string to a more human readable one using moment.js

kopijunkie committed:
Date display formatting #17

kopijunkie committed:
Merge branch 'master'

Conflicts:
	webpack.config.js

bmorelli25 commented on app/utils/dateFormatter.jsx:
```diff
@@ -0,0 +1,17 @@
+const moment = require('moment');
```
Great idea putting this in its own file

bmorelli25 reviewed (commented):
Great work again @kopijunkie! The code looks good, but I wont be able to test it on my end until tomorrow, sorry!

bmorelli25 committed:
Removing year

Removing year as it is causing line breaks on 'wednesday' due to length of name. Also, not really necessary to have the year

bmorelli25 commented:
I removed the year as it was causing line breaks on Wednesdays and isn't entirely necessary. Other than that, looks great as always @kopijunkie. Thanks for your contribution. Merging and closing.
